spec/services/csv/export_sessions_to_csv_spec.rb

```diff
@@ -51,27 +51,28 @@
 		end
 	end
 
-  it "adds a file with session notes" do
+  it "adds a file with session notes and images info" do
     session = create_session!
     note = Note.create!(
       text: "Example Note",
       date: DateTime.new(2018,8,20,11,16,44),
       latitude: BigDecimal.new("40.68038924"),
       longitude: BigDecimal.new("-73.97631499"),
+      photo: File.new("#{Rails.root}/spec/support/example_photo.jpg"),
       session: session,
     )
 
     zip_path = @subject.call([session.id])
 
     Zip::File.open(zip_path) do |zip_file|
-      actual_contents = zip_file.entries.map { |entry| entry.get_input_stream.read }
+      actual_contents = zip_file.entries.map { |entry| entry.get_input_stream.read }.last
```

Comment:
Maybe we could extract a method like `last_file_content(zip_file)` or `file_contents(zip_file).last`